### DISCUSSION TITLE
Q&A + tooltip not showing + menu problem

### DISCUSSION THREAD

**Discussion by jsupa** [beercss/beercss#285] [Q&A]
Hi, hi, I don't know what I'm doing wrong, I followed the instructions and the tooltip doesn't show up when I go to the parent :(

<img width="245" alt="image" src="https://github.com/beercss/beercss/assets/74856856/7067a4e9-c086-4a4f-8897-34f6f173d0f8">
<img width="617" alt="image" src="https://github.com/beercss/beercss/assets/74856856/61015abe-5ebb-43be-8d42-2fbd0bfe477f">
<img width="790" alt="image" src="https://github.com/beercss/beercss/assets/74856856/e41b55f1-77fb-497b-98b0-f1c8b47ee7d5">
<img width="1360" alt="image" src="https://github.com/beercss/beercss/assets/74856856/62539af1-c38e-4bff-bd10-a4b2ef83828f">

**Comment by leonardorafael** [comment]
I don't get where is the problem, but here works https://codepen.io/leo-bnu/pen/jORxeNY
Can you add a codepen to simulate it?

![image](https://github.com/beercss/beercss/assets/12979016/9a5554ef-e569-4c94-916a-eb8a9688f8cd)

**Reply by jsupa** [reply]
I found the cause and the problem was a missing doctype :(
<img width="530" alt="image" src="https://github.com/beercss/beercss/assets/74856856/71f646fe-03c8-485f-8bb1-64e2a979c9c9">

**Comment by jsupa** [comment]
And I have one more question :D its possible fix this ? 

Icon "z overflowing" ?

<img width="703" alt="image" src="https://github.com/beercss/beercss/assets/74856856/aad6bfc7-4672-4ee5-a60d-ef0e3396c1dc">
